.github/workflows/test_build_deploy.yml

```diff
@@ -219,9 +219,8 @@ jobs:
           wait_for_deployment: false
           wait_for_environment_recovery: ${{env.DEPLOYMENT_RECOVERY_TIMEOUT_SECONDS}}
 
-      - name: Push master/dev/edge Docker image
+      - name: Push dev/edge Docker image
         if: |
-          env.BRANCH_NAME == 'master' ||
```

Comment:
Mh, I think that needs to stay because our dev builds are built on commit to dev. Our staging builds are built on commit to master and production builds are created when a tag is created.

Reply:
Ah wait actually we create staging builds on tag as well. I confused it with another setup.

Reply:
You were right, but for different reasons :) - QA uses images tagged with the branch name. So, I reverted the changes in that step.